src/app/styles/charts.scss

```diff
@@ -0,0 +1,9 @@
+.chart {
+  display: block;
+  width: 300px;
```

Comment:
why chart has fixed width?

Reply:
it would be responsive of course but up to this moment when there is no piechart yet I've decided to show future view of the barchart. if float: right & fixed width remove from the charts.scss barchart just stretches within container.
float: right & fixed width will be deleted or substituted for appropriate properties

Reply:
fixed